README.md

```diff
@@ -9,11 +8,115 @@ Database synchronization utility.
 [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/laudio/sync-db.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/laudio/sync-db/context:javascript)
 [![License](https://img.shields.io/npm/l/@leapfrogtechnology/sync-db.svg)](https://github.com/kabirbaidhya/sync-db/blob/master/package.json)
 
+# Set up
+
+## Install sync-db
+
+Install `sync-db` globally.
+
+    $ yarn global add @lftechnology/sync-db
+
+Install database driver(s) of the database(s) that are to be synced.
+
+## Configure Database Connections
+
+Create `connections-sync-db.json` in your project and configure database connection(s) to be synced.
+
+```json
+{
+  "connections": [
+    {
+      "id": "db1",
+      "host": "localhost",
+      "port": 1433,
+      "user": "admin",
+      "database": "foo",
+      "password": "password",
+      "client": "mssql"
+    },
+    {
+      "id": "db2",
+      "host": "localhost",
+      "port": 1433,
+      "user": "admin",
+      "database": "bar",
+      "password": "password",
+      "client": "mysql2"
```

Comment:
We haven't tested this in mysql yet so let's keep it limited to `mssql` until we have another sample example for mysql.